buildSrc/build.gradle.kts

```diff
@@ -2,26 +2,37 @@
  * Copyright (c) 2021 D4L data4life gGmbH - All rights reserved.
  */
 
+import care.data4life.gradle.integration.dependency.d4l
+import care.data4life.gradle.integration.dependency.ensureKotlinVersion
+
 plugins {
     `kotlin-dsl`
+    id("care.data4life.gradle.integration.dependency")
 }
 
 repositories {
     gradlePluginPortal()
     mavenCentral()
-    maven("https://dl.bintray.com/data4life/maven")
+    google()
+    d4l()
 }
 
+ensureKotlinVersion("1.5.31")
```

Comment:
This can go

Reply:
Right, with Kotlin 1.7.10 and Gradle 6.5.1 this is not necessary anymore

Reply:
Removed